wbooks/src/app/components/BookItem/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {
+  View,
+  // Image,
+  Text
+} from 'react-native';
+
+import styles from './styles';
+
+export default function BookItem() {
```

Comment:
You need to pass a Book as prop, and use their props to show an image, title, author...